modules/statistics/RTPStatsCollector.js

```diff
@@ -401,6 +404,27 @@ StatsCollector.prototype._defineGetStatValueMethod = function(keys) {
     return (item, name) => itemStatByKey(item, keyFromName(name));
 };
 
+/**
+ * Obtains a stat value from given stat and converts it to a non-negative
+ * number. If the value is either invalid or negative then 0 will be returned.
+ * @param report
+ * @param {string} name
+ * @return {number}
+ */
+StatsCollector.prototype.getNonNegativeStat = function(report, name) {
+    let value = this._getStatValue(report, name);
```

Comment:
You can simplify all this as follows:

```
const value = this._getStatValue(report, name);

return Math.max(0, value | 0);
```

`value | 0` is a neat trick to coerce anything to a number or 0:

```
> undefined | 0
0
> '123' | 0
123
> null | 0
0
> -1 | 0
-1
> NaN | 0
0
> 123 | 0
123
>
```